test/e2e-v2/java-test-service/e2e-service-provider/src/main/java/org/apache/skywalking/e2e/controller/UserController.java

```diff
@@ -68,4 +80,39 @@ private long randomSleepLong(int min, int max) {
         int randomNumber = rand.nextInt((max - min) + 1) + min;
         return randomNumber;
     }
+
+    private void testCacheService() {
```

Comment:
@kezhenxu94 Do you have recommendation about the test service to include virtual cache testing?

Reply:
@pg-yang Could you share why do we need so many put/get in this method?

Reply:
I have removed some put/get  method .